Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -478,6 +482,10 @@
 ### LS2 - MWGR ###
 steps['RunCosmics2019']={'INPUT':InputInfo(dataSet='/ExpressCosmics/Commissioning2019-Express-v1/FEVT',label='2019GR3',run=[330098],events=100000,location='STD')}
 
+#### Test of lumi section boundary crossing with run2 2018D ####
+Run2018Dml={320822: [[1,1]] , 320823: [[1,1]]}
+steps['RunEGamma2018Dml']={'INPUT':InputInfo(dataSet='/EGamma/Run2018D-v1/RAW',label='2018D',events=100000,location='STD', ls=Run2018Dml)}
```

Comment:
This leads to run boundary (between runs 320822 and 320823) in addition to lumi boundary, no?

Reply:
yes, I am working to a revised version that adds a lumi boundary only workflow